CHANGELOG.md

```diff
@@ -16,6 +16,15 @@
 ### passkey-client
 
 - Changed: The `Client` no longer hardcodes the UV value sent to the `Authenticator` ([#22](https://github.com/1Password/passkey-rs/pull/22)).
+- Changed: The `Client` no longer hardcodes the RK value sent to the `Authenticator` ([#27](https://github.com/1Password/passkey-rs/pull/27)).
+- The client now supports additional user-defined properties in the client data, while also clarifying how the client
+handles client data and its hash.
+	- ⚠ BREAKING: Changed: `register` and `authenticate` take `ClientData<E>` instead of `Option<Vec<u8>>`.
+	- ⚠ BREAKING: Changed: Custom client data hashes are now specified using `DefaultClientDataWithCustomHash(Vec<u8>)` instead of
+		`Some(Vec<u8>)`.
+	- Added: Additional fields can be added to the client data using `DefaultClientDataWithExtra(ExtraData)`.
+- `CollectedClientData` is now generic and supports additional strongly typed fields.
+	- Changed: `CollectedClientData` has changed to `CollectedClientData<E = ()>`
```

Comment:
**Nit:** This is in `passkey-types` not `passkey-client`

Reply:
Moved!